apps/prairielearn/python/prairielearn/html_utils.py

```diff
@@ -119,9 +126,10 @@ def _get_attrib(
 
 
 def has_attrib(element: lxml.html.HtmlElement, name: str) -> bool:
-    """
-    Return true if the element has an attribute of that name,
-    false otherwise.
+    """If an HTML element has an attribute `name` set.
+
+    Returns:
+        True if the element has an attribute of that name, False otherwise.
```

Comment:
Should have True and False wrapped in back ticks.

Reply:
Fixed in [533ae66](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/533ae668d695d5ca5a3f241cdcd98776a7728dbf).